feature/preview/src/main/java/com/google/jetpackcamera/feature/preview/ui/CameraControlsOverlay.kt

```diff
@@ -344,16 +344,32 @@ private fun CaptureButton(
                         onCaptureImageWithUri(
                             context.contentResolver,
                             null,
-                            true,
-                            previewUiState.previewMode.onImageCapture
-                        )
+                            true
+                        ) { event: PreviewViewModel.ImageCaptureEvent, _: Int ->
+                            previewUiState.previewMode.onImageCapture(event)
+                        }
                     }
 
                     is PreviewMode.ExternalImageCaptureMode -> {
                         onCaptureImageWithUri(
                             context.contentResolver,
                             previewUiState.previewMode.imageCaptureUri,
-                            false,
+                            false
+                        ) { event: PreviewViewModel.ImageCaptureEvent, _: Int ->
+                            previewUiState.previewMode.onImageCapture(event)
+                        }
+                    }
+
+                    is PreviewMode.ExternalMultipleImageCaptureMode -> {
+                        val ignoreUri = previewUiState.previewMode.imageCaptureUris.isNullOrEmpty()
+                        onCaptureImageWithUri(
+                            context.contentResolver,
+                            null,
+                            if (previewUiState.previewMode.imageCaptureUris.isNullOrEmpty()) {
+                                true
+                            } else {
+                                ignoreUri
+                            },
```

Comment:
Wouldn't this be equivalent to:
`previewUiState.previewMode.imageCaptureUris.isNullOrEmpty() || ignoreUri`?